components/ResidentIdSearchbar/index.js

```diff
@@ -32,44 +37,43 @@ const ResidentIdSearchbar = ({ surveyee, setSurveyee, surveyingOrganization }) =
             });
           }
           const allData = residentData.concat(offlineData);
-          // console.log(allData)
-          setData(allData || []);
-          setResidents(allData.slice() || [].slice());
+          setResidentsData(allData.slice() || []);
         });
       }
       setLoading(false);
     });
   };
 
-  const fetchData = async () => {
+  const fetchOnlineData = async () => {
     setLoading(true);
-    checkOnlineStatus().then(async (connected) => {
-      if (connected) {
-        const queryParams = {
-          skip: 0,
-          offset: 0,
-          limit: 2000,
-          parseColumn: 'surveyingOrganization',
-          parseParam: surveyingOrganization,
-        };
-        const records = await residentQuery(queryParams);
-        let offlineData = [];
-        await getData('offlineIDForms').then((offlineResidentData) => {
-          if (offlineResidentData !== null) {
-            Object.entries(offlineResidentData).forEach(([key, value]) => { //eslint-disable-line
-              offlineData = offlineData.concat(value.localObject);
-            });
-          }
+    setOffline(false);
+    const queryParams = {
+      skip: 0,
+      offset: 0,
+      limit: 2000,
```

Comment:
In my recent settings PR, I created an async data point called findRecordsLimit In Settings >FindRecrds. This is where a user can set their amount on their device. Should we be using that for the limit instead of default 2000?